package.json

```diff
@@ -23,91 +23,76 @@
-    "all-contributors-cli": "4.10.1",
+    "@storybook/addon-options": "4.0.0-alpha.3",
+    "@storybook/channels": "4.0.0-alpha.3",
+    "@storybook/react": "4.0.0-alpha.3",
+    "all-contributors-cli": "5.4.0",
     "babel-cli": "6.26.0",
-    "babel-eslint": "8.2.1",
-    "babel-jest": "22.2.2",
-    "babel-plugin-flow-react-proptypes": "17.1.2",
-    "babel-plugin-lodash": "3.3.2",
-    "babel-plugin-macros": "2.1.0",
-    "babel-plugin-module-resolver": "3.1.0",
-    "babel-plugin-preval": "1.6.3",
+    "babel-eslint": "8.2.6",
+    "babel-jest": "23.4.2",
+    "babel-plugin-flow-react-proptypes": "24.1.1",
+    "babel-plugin-lodash": "3.3.4",
+    "babel-plugin-macros": "2.4.0",
+    "babel-plugin-module-resolver": "3.1.1",
+    "babel-plugin-preval": "3.0.0",
     "babel-plugin-transform-class-properties": "6.24.1",
     "babel-plugin-transform-object-rest-spread": "6.26.0",
     "babel-plugin-transform-runtime": "6.23.0",
-    "babel-preset-env": "1.6.1",
+    "babel-preset-env": "1.7.0",
     "babel-preset-react": "6.24.1",
     "babel-runtime": "6.26.0",
-    "bithound": "1.7.0",
-    "bundlesize": "0.16.0",
-    "clean-webpack-plugin": "0.1.18",
+    "bundlesize": "0.17.0",
+    "clean-webpack-plugin": "0.1.19",
     "codeclimate-test-reporter": "0.5.0",
-    "commitizen": "2.9.6",
     "condition-circle": "2.0.1",
-    "cross-env": "5.1.3",
-    "css-loader": "0.28.9",
-    "cssnano": "3.10.0",
+    "cross-env": "5.2.0",
+    "css-loader": "1.0.0",
     "cz-conventional-changelog": "2.1.0",
-    "enzyme": "3.3.0",
-    "enzyme-adapter-react-16": "1.1.1",
-    "enzyme-to-json": "3.3.1",
-    "eslint": "4.17.0",
-    "eslint-config-airbnb": "16.1.0",
-    "eslint-config-prettier": "2.9.0",
-    "eslint-plugin-flowtype": "2.44.0",
-    "eslint-plugin-import": "2.8.0",
-    "eslint-plugin-jsx-a11y": "6.0.3",
-    "eslint-plugin-react": "7.6.1",
+    "enzyme": "3.4.4",
+    "enzyme-adapter-react-16": "1.2.0",
+    "enzyme-to-json": "3.3.4",
+    "eslint": "5.4.0",
+    "eslint-config-airbnb": "17.1.0",
+    "eslint-config-prettier": "3.0.1",
+    "eslint-plugin-flowtype": "2.50.0",
+    "eslint-plugin-import": "2.14.0",
+    "eslint-plugin-jsx-a11y": "6.1.1",
+    "eslint-plugin-react": "7.11.1",
     "extract-text-webpack-plugin": "3.0.2",
-    "find-up": "2.1.0",
-    "flow-bin": "0.70.0",
-    "flow-copy-source": "1.3.0",
+    "find-up": "3.0.0",
+    "flow-bin": "0.79.1",
+    "flow-copy-source": "2.0.2",
     "html-loader": "0.5.5",
-    "husky": "0.14.3",
-    "ignore-loader": "0.1.2",
-    "jest": "22.3.0",
-    "lint-staged": "6.1.0",
-    "lodash": "4.17.5",
-    "markdown-loader": "2.0.2",
-    "nightwatch": "0.9.19",
-    "npm-run-all": "4.1.2",
-    "picturebook": "1.2.4",
-    "polished": "1.9.2",
     "postcss-cssnext": "3.1.0",
-    "postcss-discard-comments": "2.0.4",
-    "postcss-flexbugs-fixes": "3.3.0",
-    "postcss-for": "2.1.1",
-    "postcss-import": "11.1.0",
+    "postcss-import": "12.0.0",
```

Comment:
not all css could be removed since we are still using it for stories. It didn't seem worth going through the trouble of removing it entirely (it's a decent amount of work). At least this removes the unused plugins